Mage.Sets/src/mage/cards/w/WordOfCommand.java

```diff
@@ -0,0 +1,249 @@
+/*
+ *  Copyright 2010 BetaSteward_at_googlemail.com. All rights reserved.
```

Comment:
Just a heads up - no need to put the license header at the top of files any more :smile: the license file in the root of the project is enough!

Reply:
Especially since I just got rid of them all ;) https://github.com/magefree/mage/pull/4998

Reply:
Yeah I started writing this prior to that PR, so I'll have to change it accordingly.